WordPress/gutenberg
# Use nested blocks for quotes

## Description

Related: #5265.

This branch changes both quotes block so that they use nested blocks for the content. This enables the user to add content other than paragraphs and simplifies the block registration.

~~WIP, to do:~~

* ~~Preserve citation when unwrapping the blocks.~~
* ~~Make paste work.~~
* ~~Double enter should exit the quote.~~ Should this be a general nested block behaviour?
* ~~Update demo content.~~

## How Has This Been Tested?
* Ensure old quotes still work (try demo content).
* Ensure transformations work.
* Try the shortcut: `> + SPACE` both with and without existing content.

## Checklist:
